### PR TITLE
Rest api jvb integration

### PR DESCRIPTION
**Pull request by bbaldino**
This implements v1 of the new JVB API

### CODE REVIEW

File: jvb_pom.xml [hunk 1]
```diff
@@ -114,6 +114,11 @@
       <artifactId>sctp</artifactId>
       <version>1.0-20190319.172750-1</version>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>jvb-api-server</artifactId>
+      <version>2.1-227-g3ce5bf97</version>
```
Comment (JonathanLennox) [inline jvb_pom.xml:120]: Should this be ${project.version}?

File: src/main/kotlin/org/jitsi/videobridge/signaling/api/SignalingApiConfig.kt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        fun bindPort(): Int = bindPortProperty.value
+
+        class BindAddressProperty : SimpleProperty<String>(
+            newConfigAttributes {
+                readOnce()
+                name("videobridge.apis.jvb-api.bind-address")
+            }
+        )
+
+        private val bindAddressProperty = BindAddressProperty()
```
Comment (JonathanLennox) [inline src/main/kotlin/org/jitsi/videobridge/signaling/api/SignalingApiConfig.kt:53]: Do you have any desire to validate this as being a valid IP address, or do you just want to let whatever exception gets thrown at initialization handle it?

File: jvb_pom.xml (hunk 1 above)
Comment (bbaldino) [reply jvb_pom.xml:120]: Yup, thanks

File: src/main/kotlin/org/jitsi/videobridge/signaling/api/SignalingApiConfig.kt (hunk 2 above)
Comment (bbaldino) [reply src/main/kotlin/org/jitsi/videobridge/signaling/api/SignalingApiConfig.kt:53]: I think elsewhere we just let it fail further down which seems fine to me.

**Pull request merged**